exporter/elasticsearchexporter/factory.go

```diff
@@ -41,8 +38,18 @@ func NewFactory() exporter.Factory {
 }
 
 func createDefaultConfig() component.Config {
+	// TODO(lahsivjar): This is deviating from the original defaults:
```

Comment:
++. 2 concerns here:
- block_on_overflow: I incline to have it as `true` so that the default es exporter behavior is not too breaking although both `true` and `false` come with different problems. You may easily convince me the other way around.
- wait_for_result: I believe the default is `true` and that is desired. But it is not our previous default and this behavior change should be clearly stated in the changelog

Reply:
> Is that the desired end state of the defaults though?

Yes, I think `wait_for_result` is a specialized config and we should not turn it on by default.

> This will have a significant impact on latency and throughput of small serial requests

Agreed, `wait_for_result` set to `true` will have a significant impact on latency and throughput. That is why I have kept it to `false` and that is also the reason why I think `false` is a better default for this configuration option.